arch/xtensa/include/xtensa-asm2-context.h

```diff
@@ -101,11 +103,16 @@
 #define BSA_LEND_OFF	(BASE_SAVE_AREA_SIZE - 52)
 #define BSA_LCOUNT_OFF	(BASE_SAVE_AREA_SIZE - 56)
 
+#define BSA_EXCCAUSE_OFF \
+			(BASE_SAVE_AREA_SIZE - \
+			 (BASE_SAVE_AREA_SIZE_COMMON + \
+			  BASE_SAVE_AREA_SIZE_LOOPS + 4))
```

Comment:
What is `4` for? Is it `BASE_SAVE_AREA_SIZE_EXCCAUSE`?

Reply:
Yes, it is for `BASE_SAVE_AREA_SIZE_EXCCAUSE`. Have replaced it.